src/datastore/common.ts

```diff
@@ -144,13 +143,17 @@ export interface BaseTx {
   token_transfer_recipient_address?: string;
   /** 64-bit unsigned integer. */
   token_transfer_amount?: bigint;
+  // TODO(perf): use hex string since that is what we already get from deserializing event payloads
+  //   and from the pg-node adapter (all the pg js libs use text mode rather than binary mode)
   /** Hex encoded arbitrary message, up to 34 bytes length (should try decoding to an ASCII string). */
   token_transfer_memo?: Buffer;
   status: DbTxStatus;
   type_id: DbTxTypeId;
   /** Only valid for `contract_call` tx types */
   contract_call_contract_id?: string;
   contract_call_function_name?: string;
+  // TODO(perf): use hex string since that is what we already get from deserializing event payloads
+  //   and from the pg-node adapter (all the pg js libs use text mode rather than binary mode)
   /** Hex encoded Clarity values. Undefined if function defines no args. */
   contract_call_function_args?: Buffer;
   abi?: string;
```

Comment:
With the cpu optimizations from this PR, profiling indicates around a 3rd of the cpu time is spent on the postgres adapter layer (converting postgres data to and from js data). Which is great news! Because it indicates the rest of the code is now using relatively little cpu. 

One of the slow paths around the pg data layer is converting between Buffers and hex strings. The nodejs pg library we use (and the newer one we are considering to use) both operated in "text" mode rather than binary mode, which means even though many columns are stored in `bytea` format, they are still converted to a hex string before being sent to the nodejs layer, and the adapter then has to parse it into a buffer. Same thing for writing buffers from nodejs to pg, the adapter encodes Buffers to a hex string before sending the queries to pg. 

In many of these cases, we already _start off_ with a hex string and convert it into a Buffer when submitting pg queries, and when we get a Buffer from the pg adapter, we re-encode it back into a hex string to send to client http requests. So it looks like we'd cut down the cpu time a decent amount (need to do more benchmarking to know how much) by just avoiding the intermediate Buffer type all-together.